HomeWork_3/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace HomeWork_3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            CheckNumberMethod_1();
+            CheckNumberMethod_2();
+            CheckNumberMethod_3();
+        }
+        static void CheckNumberMethod_1()
+        {
+
+            Console.Write("Enter the number: ");
+            int number = Convert.ToInt32(Console.ReadLine());
+
+            if (number % 2 == 0 && number % 5 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+            }
+            else if (number % 2 == 0)
+            {
+                Console.WriteLine("tutti");
+            }
+            else if (number % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+            }
+
+            Console.WriteLine("\nPress Enter to continue\n");
+            Console.ReadLine();
+
+        }
+
+        static void CheckNumberMethod_2()
+        {
+
+            Console.Write("Enter the first number: ");
+            int number1 = Convert.ToInt32(Console.ReadLine());
+
+            Console.Write("Enter the second number (greater than the first): ");
+            int number2 = Convert.ToInt32(Console.ReadLine());
+
+            if (number1 >= number2)
+            {
+                Console.WriteLine("Error! The first number must be less than the second.");
+                return;
+            }
+
+            for (int i = number1 + 1; i < number2; i++)
```

Comment:
digits must be checked between the numbers entered from the keyboard, including the entered numbers.